packages/react/src/ReactElement.js

```diff
@@ -290,6 +290,11 @@ export function cloneAndReplaceKey(oldElement, newKey) {
  * See https://reactjs.org/docs/react-api.html#cloneelement
  */
 export function cloneElement(element, config, children) {
+
+  if (!element) {
+    throw new TypeError('Cannot clone null or undefined.');
```

Comment:
In the rest of the codebase we use a function called `invariant` for errors. That integrates with our system that replaces error messages with error codes in production builds. Could you use it here too?

```js
invariant(false, 'message')
```

Reply:
Yeah that’s really cool, I’ll put that in there